### PR TITLE
KATI-90_Patient-Appointment-Detail-Feature

### PR DESCRIPTION
**Pull request by vtnlinh95**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/view/activity/PatientAppointmentDetailsActivity.java [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        loadView();
+    }
+
+    private void loadView() {
+        lbRated = (LinearLayout) findViewById(R.id.lbRated);
+        lbRated.setVisibility(View.INVISIBLE);
+        lbExtraComment = (LinearLayout) findViewById(R.id.lbExtraComment);
+        ivPatientPicture = (ImageView) findViewById(R.id.ivPatientPicture);
+        ivPatientPicture.setOnClickListener(this);
+        appointmentEntity = loadData();
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/PatientAppointmentDetailsActivity.java]: This line should be in onCreate(), not in loadView()

File: app/src/main/java/com/kms/cura/view/activity/ViewDoctorProfileActivity.java [hunk 2]
```diff
@@ -32,8 +36,15 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     private void loadFragment() {
-        data = getIntent().getStringExtra(SearchActivity.DOCTOR_SELECTED);
-        doctorUserEntity = JsonToEntityConverter.convertJsonStringToEntity(data, DoctorUserEntity.getDoctorEntityType());
+        int position = getIntent().getIntExtra(PatientAppointmentDetailsActivity.PATIENT_POSITION, -1);
+        if(position != -1){
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/ViewDoctorProfileActivity.java]: re-format this file

**Comment by vtnlinh95** [comment]
@hieumai  fixed

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @toannguyen71195  please start verifying on this

**Comment by huuduy216** [comment]
@vtnlinh95 On tablet, the status tag cover up part of the name.
screenshot_20160726-100350

**Comment by vtnlinh95** [comment]
@huuduy216  fixed

**Comment by vtnlinh95** [comment]
@huuduy216  fixed

**Comment by hieumai** [comment]
@vtnlinh95 please resolve conflicts so I can review again

**Comment by hieumai** [comment]
@vtnlinh95 please resolve conflicts and squash commits so I can merge

**Comment by vtnlinh95** [comment]
@hieumai rebased

**Pull request merged**